spring-kafka/src/main/java/org/springframework/kafka/retrytopic/DeadLetterPublishingRecovererFactory.java

```diff
@@ -43,6 +46,8 @@
  */
 public class DeadLetterPublishingRecovererFactory {
 
+	private static final LogAccessor logger = new LogAccessor(LogFactory.getLog(RetryTopicConfigurer.class));
```

Comment:
Copy/paste artefact: wrong class is used for Log instance 😄

Reply:
Thanks for noticing - there should be a plugin that threw a CopyPasteException in these situations! :lay